### PR TITLE
f-registration@v0.1.0 - initial component structure and config

### PR DESCRIPTION
**Pull request by ashleynolan**
This is the base commit for the `f-registration` component. It is essentially just a copy of the `f-skeleton` component, with component name updates throughout.

Subsequent PR will introduce more specific component markup and styling.

### Added
- `f-registration` component structure and config

---

## UI Review Checks

- [x] README and/or UI Documentation has been created
- [x] Unit tests have been created

### CODE REVIEW

File: packages/f-registration/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ```bash
+    npm install @justeat/f-registration
+    ```
+
+2.  Import the component
+
+    You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+
+    ```
+    import VueMyComponent from '@justeat/f-registration';
```
Comment (DamianMullins) [inline packages/f-registration/README.md]: Maybe call the component something more descriptive?

**Review by DamianMullins — DISMISSED** [review]
Minor comment left, happy to approve without it though.

File: packages/f-registration/README.md (hunk 1 above)
Comment (ashleynolan) [reply packages/f-registration/README.md]: Have updated this now. Just in process of trying to fix the build…

**Review by DamianMullins — DISMISSED** [review]

File: packages/f-registration/src/demo/Index.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// Demo component for testing `f-form-field` independently via `vue serve --open src/demo/Index.vue`
```
Comment (minrwhite) [inline packages/f-registration/src/demo/Index.vue]: Should this comment reference `f-form-field`, or should this be `f-registration`; and if so is this an oversight in the Skeleton component, on which this is based?

File: packages/f-registration/src/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
+ * @overview Fozzie Form Field Component JS Wrapper
+ *
+ * @module f-form-field
```
Comment (minrwhite) [inline packages/f-registration/src/index.js]: Again, `f-form-field` or `f-registration`?

File: packages/f-registration/src/demo/Index.vue (hunk 2 above)
Comment (ashleynolan) [reply packages/f-registration/src/demo/Index.vue]: This is my fault – the component started as f-form-field and I changed it, so need to update it.

File: packages/f-registration/src/index.js (hunk 3 above)
Comment (ashleynolan) [reply packages/f-registration/src/index.js]: Good spot – will update

**Comment by ashleynolan** [comment]
@benwhite-deltas Have updated this now and fixed those references

**Review by minrwhite — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**